src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const successEl = (message) =>
+  `<div data-qa="notification" class="success">${message}</div>`;
+const errorEl = (message) =>
+  `<div data-qa="notification" class="error">${message}</div>`;
+
+const displayMessage = (message) => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', message);
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject(errorEl('First promise was rejected'));
+  }, 3000);
+
+  const handleClick = () => {
+    clearTimeout(timeoutId);
+    resolve(successEl('First promise was resolved'));
+    document.removeEventListener('click', handleClick);
+  };
+
+  document.addEventListener('click', handleClick);
+});
```

Comment:
You have used `setTimeout` and `clearTimeout` to handle the rejection of the first promise after 3 seconds. This is a valid approach, but it can be simplified by using `Promise.race` method. It accepts an array of promises and resolves/rejects as soon as one of the promises resolves or rejects. You can create a new promise that rejects after 3 seconds using `new Promise((resolve, reject) => setTimeout(() => reject('First promise was rejected'), 3000))`, and race it against the promise that resolves on click.